src/app/layout.tsx

```diff
@@ -85,6 +123,7 @@ export default function RootLayout({
               </ul>
             </div>
             <div className="navbar-end">
+              <radix-connect-button></radix-connect-button>
```

Comment:
can we pass the className to this? or how do we style it?

Reply:
I think the ability to style the button is coming with the next rcnet release, due at the end of this week.